{{cookiecutter.project_slug}}/.github/workflows/ci.yaml

```diff
@@ -2,7 +2,7 @@ name: CI
 on:
   push:
     branches:
-      - "*"
+      - "main"
```

Comment:
is this right? I was getting two runs otherwise : one for "push" and one for "pull request". The xarray config does uses `"*"` though.

Reply:
`"*"` was a catch-all for any branch since some folks don't use master or main. Setting this to `main` is fine though.